Makefile

```diff
@@ -189,6 +211,8 @@ release: tag_major_minor
 	docker push $(NAME)/node-base:$(MAJOR).$(MINOR)
 	docker push $(NAME)/node-chrome:$(MAJOR).$(MINOR)
 	docker push $(NAME)/node-firefox:$(MAJOR).$(MINOR)
+	docker push $(NAME)/node-opera:$(MAJOR).$(MINOR)
+	docker push $(NAME)/node-phantomjs:$(MAJOR).$(MINOR)
```

Comment:
Why is `node-phantomjs` here? This image was deprecated a while ago.

Reply:
Deleted.